kb/2022-03-10/multiple_nics_vm_connectivity.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Multiple NICs VM Connectivity
+descripion: How to deal VMs with multiple NICs in Harvester
+slug: multiple-nics-vm-connectivity
+authors:
+  - name: Date Huang
+    title: Software Engineer
+    url: https://github.com/tjjh89017
+    image_url: https://github.com/tjjh89017.png
+tags: [vm, network]
+hide_table_of_contents: false
+---
+
+# Multiple NICs VM Connectivity
+
+## What will happen in VM with multiple NICs by default in Harvester
+
+In some scenarios, we will setup two or more NICs for VM network to serve the different purpose. If setting up DHCP server and client by default in all networks, we might get connectivity issue randomly in VM. After some reboot, it might come back with expected connection, but it still will lost connection randomly after some period.
+
+## Identify your connectivity issue that is covered in this knowledge base
+
+We take Linux VM and `iproute2` as example to identify your default route. If you cannot execute the commnad below, please switch root user or execute via `sudo` and execute again.
+
+In your VM, execute following command in your VM in normal state:
+```bash
+ip route show default
+```
+
+In normal state, you will get only one route with destation to default and show the correct gateway IP.
+```
+default via <Gateway IP> dev eth0 proto dhcp src <VM IP> metric 100
+```
+
+For example:
+```
+$ ip route show default
+default via 192.168.0.254 dev eth0 proto dhcp src 192.168.0.100 metric 100
+$
+```
+
+If issue that covered by this KB occurs, you could login VM via Web Console in Harvester Web UI and execute following command in your abnormal VM:
+```bash
+ip route show default
+```
+
+In abnormal state, you will get one default route with incorrect gateway IP.
+For example:
+```
+default via <Incorrect Gateway IP> dev eth0 proto dhcp src <VM's IP> metric 100
+```
+
+## Why cause connectivity issue randomly in VM.
```

Comment:
Why the connectivity issue occurs randomly in VM

Reply:
```suggestion
## Why the connectivity issue occurs randomly in VM
```